pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "rangehttpserver"
+version = "1.4.0"
+description = ""
+authors = ["Noah Parker <nparker@cyvl.ai>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+pytest = "^7.4.2"
+pytest-cov = "^4.1.0"
+coveralls = "^3.3.1"
+requests-toolbelt = "^1.0.0"
```

Comment:
Could you unpin these dependencies? Although this is more of an "application" than a "library", it's still likely to be used as a library. As such, [we want to avoid pinning downstream users without good reason](https://iscinumpy.dev/)

Reply:
Additionally, are any of these actually needed for runtime besides the Python version?